### ISSUE TITLE
Support the `background-size` attribute on mj-hero component

### ISSUE THREAD

**Issue by am1rb** [mjmlio/mjml#2432]
**Is your feature request related to a problem? Please describe.**
I want to have a responsive thumbnail image with a play icon on top, and this is my code:

```mjml
<mjml>
  <mj-body>
    <mj-hero background-url="https://i.ytimg.com/vi/vUNb4dXTRQs/hqdefault.jpg" background-color="#666666" padding="29% 0px">
      <mj-image width="96px" padding="0" src="https://i.ibb.co/mbM3Sxn/play.png" href="https://www.youtube.com/watch?v=vUNb4dXTRQs"></mj-image>
    </mj-hero>
  </mj-body>
</mjml>
```

And the output is something like this on desktop and mobile views:

![image](https://user-images.githubusercontent.com/573937/156146747-47c1e0eb-9fea-4d48-825e-be137138e13e.png)

![image](https://user-images.githubusercontent.com/573937/156146815-c23f7810-9c38-4d75-bdc3-6410c9cbb026.png)

If you check the mobile screenshot, some part of the image is cropped, and it is not visible on the output because the mj-image component tries to display the background image with `background-size: cover` style. I see the style on this line of code:
https://github.com/mjmlio/mjml/blob/master/packages/mjml-hero/src/index.js#L162

**Describe the solution you'd like**
It would be useful if the `mj-image` component supports the `background-size` attribute. So I can override its default value by passing `contain` in my case to have this ideal output:

![image](https://user-images.githubusercontent.com/573937/156149425-a2ef073a-a478-45e7-a42c-4c57747f97b9.png)

![image](https://user-images.githubusercontent.com/573937/156149660-120cde70-352b-475d-80c4-49c0f3cd624e.png)

As you see, the image is totally visible, and the background color is used to fill the other parts.

For now, there is no standard way to set another value for the `background-size`, so I did the below hack to make it possible:

```mjml
<mj-hero background-position="center center; background-size: contain" />
```

I have checked the above hack on different email inboxes and it works perfectly. Any suggestions or advice?

**Comment by iRyusa** [comment]
Did you tried with a css class to override this (with inline) ?

**Comment by am1rb** [comment]
@iRyusa Thank you for the quick response.

I can not use mj-style components because of some limitations on my side. But totally you are right. I just checked my results on https://app.emailonacid.com/ again and saw that the target mj-image components are broken on Outlook 2010/2013 clients because of the `background-size: contain` style:

![image](https://user-images.githubusercontent.com/573937/156185508-6ffa2b99-8414-433e-9380-a2cb73adfe94.png)

I'm gonna use an HTML conditional statement for fixing the problem on Outlook. Will keep you posted if I found something. Thanks
